### DISCUSSION TITLE
Add a "dust filter" to exclude transactions below a certain amount from showing up in Transaction History UI

### DISCUSSION THREAD

**Discussion by jon-lee** [Chia-Network/chia-blockchain#10227] [Ideas]
Some wallets are being dusted with 0.0000001 XCH transactions, which causes the transaction history in the UI to be difficult to use. A basic filter can be added to optionally hide these.

**Comment by mikewubox** [comment]
me too.

**Comment by tangdasan** [comment]
me too.

**Comment by cuigl** [comment]
me too!

**Comment by Gooseboy2234** [comment]
Me too!!!!

**Comment by brianwfreeman** [comment]
A filter would be great. Also an export to CSV option.

**Comment by wv-krmkz** [comment]
Yes. Filtering those pesky dust transactions would be very welcome! Better yet, how about a simple filtering system to customize the transaction view in the wallet by various criteria, such as: > or < any Tx size. Or by date range. Or... +++

**Comment by alexandr-efimov** [comment]
+1
